venia-integration-tests/cypress/plugins/index.js

```diff
@@ -0,0 +1,21 @@
+/// <reference types="cypress" />
+// ***********************************************************
+// This example plugins/index.js can be used to load plugins
+//
+// You can change the location of this file or turn off loading
+// the plugins file with the 'pluginsFile' configuration option.
```

Comment:
Do we need this file at this time?

Reply:
Same with `cypress/support` files

Reply:
Yeah you are right, not sure how I missed deleting this folder. Thanks Andy.